AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -2164,12 +2169,33 @@ public static void clearUserContent() {
     VarCache.clearUserContent();
   }
 
-  private static void parseSdkCounters(JSONObject response) {
+  @VisibleForTesting
+  public static HashSet<String> parseSdkCounters(JSONObject response) {
     JSONArray enabledCounters = response.optJSONArray(
             Constants.Keys.ENABLED_COUNTERS);
+    HashSet<String> counterSet = new HashSet<>();
```

Comment:
so going by the previous logic, this line should be rewritten as:
`Set<String> counterSet = new HashSet<>();`
so we're still using `HashSet` in the implementation, but if needed it could change to a different `Set` type